docs/notebooks/binning_slowly_changing_sources.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>More explanation needed here. This jumps somewhat and assumes that user know about compute and lazy evaluation in ensamble objects. If they do not know, they could wonder where did this come from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lsstseries/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I added more.